network.py

```diff
@@ -54,7 +54,7 @@ def run(self):
         counter = 1
         for time, client, content in self.workload:
             if not counter%1000:
-                sys.stdout.write('\r{0:.2f}%'.format(100*counter/float(self.N_MEASURED_REQUESTS)))
+                sys.stdout.write('\r{:.2%}'.format(counter/float(self.N_MEASURED_REQUESTS)))
```

Comment:
The % formatter will take the input, multiple by 100, then show like a float with a '%' character on the end.